ops/services/pagerduty/versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    pagerduty = {
+      source = "pagerduty/pagerduty"
+    }
+  }
```

Comment:
Sweet, a PagerDuty provider! I don't think we used that on ReportStream. Good to know.

Reply:
How is this authenticating? env var `PAGERDUTY_TOKEN`?

Reply:
Yes, the environment variable seemed like the best fit.